README.md

```diff
@@ -68,14 +68,33 @@ for NetworkManager, a connection can only be active at one device at a time.
   Note that here too the name doesn't specify the `DEVICE` but a filename. As a consequence
   `'/'` is not a valid character for the name.
 
-### `state`
+### `state` and `persistent_state`
+
+Each connection profile can have a runtime state, represented by the `state`
+setting and a persistent state, represtented by the `persistent_state` setting.
+
+The optional `state` setting supports the following values:
+
+- `up`
+- `down`
+
+It defines whether the profile is activated (`up`) or deactivated (`down`). If
+it is unset, the profile's runtime state will not be changed.
+
+The `interface_state` setting is either `present` (default) or `absent`. If the
+connection profile contains a `type` setting, the profile will be created or
+updated if the `interface_state` is `present`. If the profile is incomplete,
```

Comment:
ditto. (also should probably read "if `persistent_state` is `present`" or "if the `persistent_state` setting is `present`".)

Reply:
done